.github/workflows/dependency-metrics.yml

```diff
@@ -0,0 +1,23 @@
+name: commcare-hq dependency metrics
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  collect-metrics:
+    runs-on: ubuntu-latest
+    env:
+      REPO: commcare-hq
+      DATADOG_APP_KEY: ${{ secrets.DATADOG_APP_KEY }}
+      DATADOG_API_KEY: ${{ secrets.DATADOG_API_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: install dev-requirements
+        run: pip install -r requirements/dev-requirements.txt
```

Comment:
Do we really need all dev requirements? That could take a while to install. Would it work to use `pip` to install just the `dependency-metrics` package (its sub-dependencies should be automatically installed)?

Reply:
This is for `pip list` to work as expected. Joel did have the option of accepting a file as input instead of the output of pip list, which could be the requirements.txt file instead, but since I didn't see the use at the time and didn't port it over, this will have to do for now.

An improvement can definitely be to add that functionality to `dependency-metrics`. On the bright side, this will only run once a day